<launch>
	<node pkg="plot_navigation" name="plot_navigation" type="plot_navigation.py" output="screen"> 
		<param name="rate" value="10.0"/>
		<param name="path_topic" value="/move_base/SBPLLatticePlanner/plan"/>
		<param name="robot_front_frame" value="axesFront"/>
		<param name="robot_rear_frame" value="axesRear"/>
		<param name="reference_frame" value="map"/>
	</node>
</launch>
